lib/views/setting/lnlambda_setting_view.dart

```diff
@@ -41,7 +41,7 @@ class _LnlambdaSettingViewState extends State<LnlambdaSettingView> {
           ),
           const Text("Lambda Server"),
           TextFormField(
-            controller: TextEditingController(text: setting.lambdaServer ?? ''),
+            controller: TextEditingController(text: setting.lambdaServer),
```

Comment:
```suggestion
            controller: TextEditingController(text: setting.lambdaServer),
```

we can use another component? maybe a PopupMenuButton https://api.flutter.dev/flutter/material/PopupMenuButton-class.html or something similar?